spec/emittr_spec.rb

```diff
@@ -6,11 +6,14 @@
   describe '#on' do
     it 'should add callback to listeners list' do
       callback = Proc.new {}
+      callback_inst = Emittr::Callback.new(&callback)
+
+      allow(Emittr::Callback).to receive(:new).and_return(callback_inst)
```

Comment:
it isn't because inside the method a new instance of `Callback` is created, so we can't verify if the callback were added properly